.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   test:
     working_directory: /go/src/github.com/Azure/acs-engine
     docker:
-      - image: microsoft/acs-engine:latest
```

Comment:
Looks we are s/microsoft/acs-engine:latest/quay.io/deis/go-dev:v1.2.0/g.

The go-dev images is 
a superset of acs-engine image? Then we need to make sure go-dev can keep track of acs-engine.
or they are totally different. Then we are good.